recipe/build.sh

```diff
@@ -30,13 +30,24 @@ TEST_CMD="eval cd tests && make check-local && cd -"
 #
 # We build 3 different versions of fftw:
 #
+if [[ "$target_platform" == "linux-64" ]] || [[ "$target_platform" == "linux-32" ]] || [[ "$target_platform" == "osx-64" ]]; then
+  ARCH_OPTS_SINGLE="--enable-sse --enable-sse2 --enable-avx"
+  ARCH_OPTS_DOUBLE="--enable-sse2 --enable-avx"
+  ARCH_OPTS_LONG_DOUBLE="--enable-long-double"
+fi
+
+if [[ "$target_platform" == "ppc64le" ]]; then
+  ARCH_OPTS_SINGLE="--enable-alitvec --enable-vsx"
```

Comment:
typo, should be:
--enable-altivec

Reply:
Yup. I've actually removed that altogether for now. VSX does single precision, but I'm getting errors in the tests with either altivec or vsx on sp.

I also may disable the long double support on ppc64le since that adds a lot to the build time and the travis CI has a tim limit.